src/Lean/Parser/Term.lean

```diff
@@ -251,6 +278,12 @@ and solved by typeclass inference of the specified class.
 -/
 def instBinder := ppGroup <| leading_parser
   "[" >> withoutPosition (optIdent >> termParser) >> "]"
+/-- A `bracketedBinder` matches any kind of binder group with an explicit bracketing syntax:
```

Comment:
What work is the word "explicit" doing here? It is slightly confusing because on the next line "explicit" of course means something else.

Reply:
It is being contrasted against one of the other binder syntax categories that also allows plain identifiers (e.g. in `fun x => ...`), which is an unbracketed binder group of size 1. I think of it as similar to the `rw [e1, e2]` vs `rw e1` situation in lean 3, the brackets are optional if there is only one thing in the group.